lib/gpu/Cargo.toml

```diff
@@ -16,11 +16,13 @@ gpu = [
     "dep:gpu-allocator",
     "dep:shaderc",
 ]
+testing = []
 
 [dependencies]
 ash = { version = "0.38.0", optional = true, default-features = false, features = ["loaded", "debug"] }
 gpu-allocator = { version = "0.27.0", optional = true }
 shaderc = { version = "0.8.3", optional = true, features = ["build-from-source"]}
+lazy_static = "1.5.0"
```

Comment:
I suggest to use the standard library approach instead.
https://github.com/rust-lang-nursery/lazy-static.rs/issues/214

Reply:
Thanks, done!